src/main/java/org/spongepowered/api/GameRegistry.java

```diff
@@ -398,6 +410,97 @@
     Collection<String> getDefaultGameRules();
 
     /**
+     * Gets the {@link Statistic} with the specified id.
+     *
+     * @param id The id of the statistic to return
+     * @return The statistic with the given id or Optional.absent() if not found
+     */
+    Optional<Statistic> getStatistic(String id);
+
+    /**
+     * Gets the {@link Statistic} for the given {@link EntityStatisticType} and
+     * {@link EntityType}.
+     *
+     * @param statisticType The type of statistic to return
+     * @param entityType The entity type for the statistic to return
+     * @return The entity statistic matching the filter or Optional.absent() if
+     *         not found
+     */
+    Optional<EntityStatistic> getEntityStatistic(EntityStatisticType statisticType, EntityType entityType);
+
+    /**
+     * Gets the {@link Statistic} for the given {@link ItemStatisticType} and
+     * {@link ItemType}.
+     *
+     * @param statisticType The type of statistic to return
+     * @param itemType The item type for the statistic to return
+     * @return The item statistic matching the filter or Optional.absent() if
+     *         not found
+     */
+    Optional<ItemStatistic> getItemStatistic(ItemStatisticType statisticType, ItemType itemType);
+
+    /**
+     * Gets the {@link Statistic} for the given {@link TeamStatisticType} and
+     * team's {@link TextColor}.
+     *
+     * @param statisticType The type of statistic to return
+     * @param teamColor The team's color for the statistic to return
+     * @return The team statistic matching the filter or Optional.absent() if
+     *         not found
+     */
+    Optional<TeamStatistic> getTeamStatistic(TeamStatisticType statisticType, TextColor teamColor);
+
+    /**
+     * Gets a list of all available {@link Statistic}s that are not grouped.
+     *
+     * @return An immutable collection containing all statistics in registry
+     *         that are not grouped
+     * @see Statistics
+     */
+    Collection<Statistic> getSimpleStatistics();
```

Comment:
This method is obsolete now. They can use `Collection<Statistic> getStatistics(StatisticType statisticType)` as an alternative.